include/glow/Partitioner/Partitioner.h

```diff
@@ -100,6 +140,9 @@ class Partitioner {
   /// The cost model related to device.
   const std::vector<DeviceInfo> &deviceInfo_;
 
+  /// The backend pointers.
+  std::vector<Backend *> backends_;
```

Comment:
Do you need a copy of the backends vector here? Or would `const std::vector<Backend*> &backends_` be enough?

Reply:
Actually, this one is only for test -- passing the "mock" backends.  If we have a better idea on how to create "mock" backends in unit test by using backendKind, no need to keep this field.